app/workers/medical_safety_alert_check_worker.rb

```diff
@@ -0,0 +1,24 @@
+class MedicalSafetyAlertCheckWorker < ApplicationWorker
```

Comment:
Now both works are pretty much the same, we could either combine both into one worker, or keep distinct workers but use a parent class to DRY up the code (+ specs) and just pass through the alert type.

Reply:
Yeah, I kept thinking this, it made sense when they were checking different feeds, but now not so much. Since we can add params in the sidekiq schedule, single class makes sense.